cmd/maya-apiserver/app/server/volume_endpoint.go

```diff
@@ -21,18 +21,15 @@ const (
 // VolumeSpecificRequest is a http handler implementation. It deals with HTTP
 // requests w.r.t a single Volume.
 func (s *HTTPServer) volumeSpecificRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	// check the feature gate & switch if enabled
+	if util.CASTemplateFeatureGate() {
+		return s.volumeV1alpha1SpecificRequest(resp, req)
+	}
+
 	glog.Infof("received volume request: method '%s'", req.Method)
```

Comment:
The above message will no longer be logged. If there is a log in s.volumeV1alpha1SpecificRequest(resp, req), then this comment can be ignored.